src/main/features/dashboard/views/macro/claimStatus/helpWithFees.njk

```diff
@@ -141,21 +141,21 @@
   {% endset %}
   <p>{{ t('You will need your claim reference number: {{ ccdId }}', { ccdId: hyphenatedCcdCaseId }) }}</p>
 
-  {% if claim.claimData.interest.lastInterestCalculationDate %}
+  {# {% if claim.claimData.interest.lastInterestCalculationDate %}
     <p>{{ t('The full claim fee shown on the dashboard is calculated as on {{ createdAt }}.', { createdAt: claim.claimData.interest.lastInterestCalculationDate | date }) | safe }}</p>
   {% elif claim.claimData.interest.lastInterestCalculationDate === undefined %}
     <p>{{ t('The full claim fee shown on the dashboard is calculated as on {{ createdAt }}.', { createdAt: claim.createdAt | date }) | safe }}</p>
-  {% endif %}
+  {% endif %} #}
 </div>
-<div class="status-block">
+{# <div class="status-block">
```

Comment:
If this code is no longer required (removed as part of your story), please remove it instead of commenting it.